tests/lib/tools/report-mongodb

```diff
@@ -17,6 +17,7 @@ def _make_parser():
                         **environ_or_required('MONGO_PORT'))
     parser.add_argument('--db-name', help="Name for the database used to insert")
     parser.add_argument('--db-collection', help="Name for the collection used to insert")
+    parser.add_argument('--metadata', help="Matedata to be included in the usue")
```

Comment:
```suggestion
    parser.add_argument('--metadata', help="Matedata to be included in the insert")
```

Reply:
does this argument as well as --db-name and --db-collect need to be marked as required?

Reply:
@bboozzoo those are not needed to be required because it is possible to use env vars instead